src/api/Nhs.Appointments.Core/BookingsService.cs

```diff
@@ -188,4 +189,15 @@ public async Task RecalculateAppointmentStatuses(string site, DateOnly day)
             }
         }
     }
+
+    public async Task OrphanAppointments(string site, DateTime from, DateTime until)
+    {
+        var bookingsInSession = (await GetBookings(from, until, site))
+            .Where(b => b.Status is not AppointmentStatus.Cancelled and not AppointmentStatus.Orphaned);
+
+        foreach (var booking in bookingsInSession)
+        {
+            await SetBookingStatus(booking.Reference, AppointmentStatus.Orphaned);
+        }
+    }
```

Comment:
I don't think we should need a new method for this, you should just be able to call `RecalculateAppointmentStatuses` and it'll sort everything for you. 

Similarly though, here you're just orphaning everything but we should be re-scheduling appointments if possible. 
e.g. 2 simultaneous sessions are running with 1 capacity each (so 2 capacity per slot total). 1 appointment is booked into the first session, then that first session is cancelled. Rather than orphaning that appointment it should remain scheduled because it can be fulfilled by the 2nd session